repository/src/main/java/uk/gov/hmcts/ccd/definition/store/repository/model/CaseType.java

```diff
@@ -162,4 +164,8 @@ public void setSearchAliasFields(List<SearchAliasField> searchAliasFields) {
     public List<SearchParty> getSearchParties() {
         return searchParties;
     }
+
+    public List<Category> getCategories() {
+        return categories;
+    }
 }
```

Comment:
this class has all end 2 end  test for CaseType at  service layer.  could you add one test for  CaseTypeServiceImpl.findByCaseTypeId (). that return a Optional<CaseType> caseType   with categories ?    when(dtoMapper.map(caseTypeEntity)).thenReturn(caseType); your caseType with categorie